src/main/java/com/cloudbees/plugins/credentials/impl/BaseStandardCredentials.java

```diff
@@ -145,11 +154,16 @@ private static FormValidation checkForDuplicates(String value, ModelObject conte
                 }
                 ModelObject storeContext = store.getContext();
                 for (Domain domain : store.getDomains()) {
-                    if (CredentialsMatchers.firstOrNull(store.getCredentials(domain), CredentialsMatchers.withId(value))
-                            != null) {
+                    Credentials match = CredentialsMatchers
+                            .firstOrNull(store.getCredentials(domain), CredentialsMatchers.withId(value));
+                    if (match != null) {
                         if (storeContext == context) {
                             return FormValidation.error("This ID is already in use");
                         } else {
+                            if (match.getScope() == CredentialsScope.SYSTEM) {
+                                // system scope is not exported to child contexts
+                                continue;
```

Comment:
🐛 Should it continue with other credentials in the domain then? firstOrNull skips others from what I see